models/api.model.js

```diff
@@ -42,4 +44,33 @@ exports.selectCommentsByArticleId =  (article_id)=>{
             return Promise.reject({status: 404, msg: '404 Not Found'})
         }
     })
+}
+
+exports.insertNewComment= (article_id, newComment)=>{
+     return db.query('SELECT * FROM comments WHERE article_id = $1', [article_id])
```

Comment:
We have a model that selects articles by article id already, we can reutilise it 🙂